cpu/nrf51/vectors.c

```diff
@@ -72,56 +72,56 @@ WEAK_DEFAULT void isr_swi5(void);
 /* interrupt vector table */
 ISR_VECTORS const void *interrupt_vector[] = {
     /* Exception stack pointer */
-    (void*) (&_estack),             /* pointer to the top of the stack */
+    __extension__ (void*) (&_estack),             /* pointer to the top of the stack */
```

Comment:
What the [commit message](https://github.com/OlegHahm/RIOT/commit/9fed4a68ea3b6d5edbc5faed8ace5c4a46a63c1c) says.